members/templates/alumni_signup_email_admin.html

```diff
@@ -1,11 +1,5 @@
 {% autoescape off %}
-    Namn: {{ alumni.name }}
-    Email: {{ alumni.email }}
-    Tel nr: {{ alumni.phone_number }}
-    Inskrivningsår: {{ alumni.year_of_admission }}
-    Arbetsplats: {{ alumni.employer }}
-    Arbetsuppgift: {{ alumni.work_title }}
-    Adress: {{ alumni.address }}
-    TFiF medlemskap: {{ alumni.tfif_membership }}
-    Får kontaktas: {{ alumni.alumni_newsletter_consent }}
+    {{ alumni.name }} har registrerat sig till ARG.
+
+    Full information: https://datateknologerna.org/admin/members/alumnisignup/{{ alumni_id }}
```

Comment:
This URL does not seem to exist in our URLS?
Or do we have a issue to create it?

Reply:
Oh duh right it was admin msg and page 😄 Yeah should be good